x-pack/legacy/plugins/ml/public/application/data_frame_analytics/common/analytics.ts

```diff
@@ -128,13 +150,17 @@ export const getPredictionFieldName = (analysis: AnalysisConfig) => {
   return predictionFieldName;
 };
 
-export const getPredictedFieldName = (resultsField: string, analysis: AnalysisConfig) => {
+export const getPredictedFieldName = (
```

Comment:
I tried adding a custom `prediction_field_name`. The job seems to run ok, but the evaluate panel fails to populate, and the predicted field doesn't show by default in the table (I do see it in the fields popup):

![image](https://user-images.githubusercontent.com/7405507/70641286-513cb980-1c35-11ea-8b31-120ff376bbcf.png)

Reply:
Thanks, @peteharverson 🙏 Got the fix in here: 55728eacc79755208e46abca8acf4c1ffbbb4274